src/popup/useApplier.ts

```diff
@@ -0,0 +1,20 @@
+import { useCallback, useEffect, useRef } from 'react';
+
+// when apply function is called
+// applyRequired will be true next render
+// it's used for syncing state with storage/localStorage only in certain moments
```

Comment:
This would be better as documentation comment, since this would be shown in IDEs as fn docs.

Reply:
Ok